modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExecutionServiceImpl.java

```diff
@@ -1022,17 +1022,20 @@ private void enlist(int tableId, List<NodeWithTerm> assignments) {
                     for (int p = 0; p < partsCnt; p++) {
                         TablePartitionId tablePartId = new TablePartitionId(tableId, p);
 
-                        NodeWithTerm enlistmentToken = assignments.get(p);
+                        NodeWithEnlistmentToken enlistmentToken = assignments.get(p);
```

Comment:
Could you please also rename the `enlistmentToken` to node or similar? `enlistmentToken.enlistmentConsistencyToken()` looks confusing.